src/pkgcheck/checks/repo_metadata.py

```diff
@@ -688,3 +688,35 @@ def feed(self, pkgs):
         pkg = pkgs[0]
         yield from self._conflicts(pkg)
         yield from self._matching(pkg)
+
+
+class EmptyProject(results.Warning):
+    """A project has no developers."""
+
+    def __init__(self, project):
+        super().__init__()
+        self.project = project
+
+    @property
+    def desc(self):
+        return f"{self.project!r} project has no members"
+
+
+class ProjectMetadataCheck(Check):
+    """Check projects.xml for issues."""
+
+    scope = base.repository_scope
+    _source = sources.EmptySource
+    known_results = frozenset([
+        EmptyProject,
+    ])
+
+    def __init__(self, *args):
+        super().__init__(*args)
+        self.repo = self.options.target_repo
+
+    def finish(self):
+        for key, project in self.repo.projects_xml.projects.items():
+            if not project.recursive_members:
+                # TODO: make it serializable
```

Comment:
What's this comment in regards to? Making project object instances serializable?

Reply:
Hmm, I can't reproduce now. I'm gonna repush and see what happens.

Reply:
You can see the error in travis now. Lemme know if I should fix the Project instance somehow or you want to fix it on `to_json` end instead.